apps/src/templates/CensusFollowUp.jsx

```diff
@@ -0,0 +1,117 @@
+import React, {Component} from 'react';
+import Checkbox from './forms/Checkbox';
+import color from "../util/color";
+import i18n from "@cdo/locale";
+
+const styles = {
+  question: {
+    fontSize: 16,
+    fontFamily: '"Gotham 5r", sans-serif',
+    color: color.charcoal,
+    paddingTop: 10,
+    paddingBottom: 5
+  },
+  option: {
+    fontFamily: '"Gotham 4r", sans-serif',
+    color: color.charcoal,
+  },
+  options: {
+    marginLeft: 35
+  },
```

Comment:
What you have here is fine, however I'd recommend calling this `styles.checkboxOptions`, to disambiguate from `styles.option`